server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -331,14 +334,22 @@ private Modal renderDownloadApplicationsModal(
   }
 
   private DivTag renderApplicationListItem(
-      Application application, boolean displayStatus, Optional<Boolean> maybeEligibilityStatus) {
+      Application application,
+      boolean displayStatus,
+      Optional<Boolean> maybeEligibilityStatus,
+      Optional<StatusDefinitions.Status> defaultStatus) {
     String applicantNameWithApplicationId =
         String.format(
             "%s (%d)",
             applicantUtils.getApplicantNameEnUs(application.getApplicantData().getApplicantName()),
             application.id);
     String viewLinkText = "View →";
 
+    String statusString = application.getLatestStatus().orElse("None");
+    statusString +=
+        defaultStatus.isPresent() && statusString.equals(defaultStatus.get().statusText())
```

Comment:
Definitely like the `matches` idea, but it doesn't like that `statusString` is non-final and used inside a lambda.  I also realized this may do something funky if you actually have a status named "None", which is perhaps not outside the realm of possibility.

This is not terribly pretty, but I think it ends up being more correct.
```
Optional<String> appStatusString = application.getLatestStatus();
    String statusString;
    if (appStatusString.isPresent()) {
      statusString =
          appStatusString.get()
              + (defaultStatus
                      .map(
                          defaultStatusString -> defaultStatusString.matches(appStatusString.get()))
                      .orElse(false)
                  ? " (default)"
                  : "");
    } else {
      statusString = "None";
    }
```